packages/fax/tests/rest/v3/services/services.steps.ts

```diff
@@ -0,0 +1,180 @@
+  listResponse = await servicesApi.list({
+    pageSize: 2,
+  });
+});
+
+Then('the response contains {string} services', (expectedAnswer: string) => {
+  const expectedServices = parseInt(expectedAnswer, 10);
+  assert.equal(listResponse.data.length, expectedServices);
+});
+
+When('I send a request to list all the services', async () => {
+  for await (const service of servicesApi.list({ pageSize: 2 })) {
+    servicesList.push(service);
+  }
+});
+
+Then('the services list contains {string} services',  (expectedAnswer: string) => {
+  const expectedServices = parseInt(expectedAnswer, 10);
+  assert.equal(servicesList.length, expectedServices);
+});
+
+When('I send a request to retrieve a service', async () => {
+  service = await servicesApi.get({
+    serviceId: '01W4FFL35P4NC4K35FAXSERVICE',
+  });
+});
+
+Then('the response contains a service object', () => {
+  assert.equal('01W4FFL35P4NC4K35FAXSERVICE', service.id);
+  assert.equal('application/json', service.webhookContentType);
+  assert.equal('+12014444444', service.defaultFrom);
+  assert.equal(2, service.numberOfRetries);
+  assert.equal(30, service.retryDelaySeconds);
+  assert.equal('MONOCHROME', service.imageConversionMethod);
+  assert.equal('123coffee-dada-beef-cafe-baadc0de5678', service.projectId);
+  assert.equal(false, service.defaultForProject);
+  assert.equal(true, service.saveInboundFaxDocuments);
+  assert.equal(true, service.saveOutboundFaxDocuments);
+  assert.equal('Fax service for e2e tests', service.name);
+  assert.equal('https://my-callback-server.com/fax', service.incomingWebhookUrl);
+});
+
+When('I send a request to update a service', async () => {
+  service = await servicesApi.update({
+    serviceId: '01W4FFL35P4NC4K35FAXSERVICE',
+    updateServiceRequestBody: {
+      name: 'Updated Fax service name',
+      webhookContentType: 'multipart/form-data',
+      defaultForProject: true,
+      numberOfRetries: 3,
+      retryDelaySeconds: 60,
+      imageConversionMethod: 'HALFTONE',
+      saveOutboundFaxDocuments: false,
+      saveInboundFaxDocuments: false,
+    },
+  });
+});
+
+Then('the response contains a service with updated parameters', () => {
+  assert.equal('01W4FFL35P4NC4K35FAXSERVICE', service.id);
+  assert.equal('multipart/form-data', service.webhookContentType);
+  assert.equal(3, service.numberOfRetries);
+  assert.equal(60, service.retryDelaySeconds);
+  assert.equal('HALFTONE', service.imageConversionMethod);
+  assert.equal(true, service.defaultForProject);
+  assert.equal(false, service.saveInboundFaxDocuments);
+  assert.equal(false, service.saveOutboundFaxDocuments);
+  assert.equal('Updated Fax service name', service.name);
+});
+
+When('I send a request to remove a service', async () => {
+  deleteServiceResponse = await servicesApi.delete({
+    serviceId: '01W4FFL35P4NC4K35FAXSERVICE',
+  });
+});
+
+Then('the delete service response contains no data', function () {
```

Comment:
What about a test to request for deleted/unknown service ?

Reply:
Sure, we can add more use cases to improve our coverage!